src/EventGrid/EventGrid/help/Get-AzEventGridSystemTopic.md

```diff
@@ -0,0 +1,158 @@
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Name
+EventGrid topic name.
+
+```yaml
+Type: String
+Parameter Sets: SystemTopicNameParameterSet
+Aliases: SystemTopicName
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -NextLink
+The link for the next page of resources to be obtained.
+This value is obtained with the first Get-AzEventGrid cmdlet call when more resources are still available to be queried.
+
+```yaml
+Type: String
+Parameter Sets: NextLinkParameterSet
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ODataQuery
+The OData query used for filtering the list results.
+Filtering is currently allowed on the Name property only.The supported operations include: CONTAINS, eq (for equal), ne (for not equal), AND, OR and NOT.
+
+```yaml
+Type: String
+Parameter Sets: ResourceGroupNameParameterSet, SystemTopicNameParameterSet
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+The name of the resource group.
+
+```yaml
+Type: String
+Parameter Sets: ResourceGroupNameParameterSet, SystemTopicNameParameterSet
+Aliases: ResourceGroup
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Top
+The maximum number of resources to be obtained.
+Valid value is between 1 and 100.
+If top value is specified and more results are still available, the result will contain a link to the next page to be queried in NextLink.
+If the Top value is not specified, the full list of resources will be returned at once.
+
+```yaml
+Type: Int32
+Parameter Sets: ResourceGroupNameParameterSet, SystemTopicNameParameterSet
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### System.String
+
+### System.Nullable`1[[System.Int32, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]
+
+## OUTPUTS
+
+### Microsoft.Azure.Commands.EventGrid.Models.PSSystemTopic
+
+### Microsoft.Azure.Commands.EventGrid.Models.PSSytemTopicListInstance
+
+## NOTES
+
+## RELATED LINKS
```

Comment:
It's nice to have if we can provide relate links.

Reply:
fixed